DFC.ServiceTaxonomy.VersionComparison/Controllers/AdminController.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DFC.ServiceTaxonomy.VersionComparison.Models;
+using DFC.ServiceTaxonomy.VersionComparison.Services;
+using DFC.ServiceTaxonomy.VersionComparison.ViewModels;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Localization;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using OrchardCore.ContentManagement;
+using OrchardCore.DisplayManagement;
+using OrchardCore.DisplayManagement.ModelBinding;
+using OrchardCore.DisplayManagement.Notify;
+
+namespace DFC.ServiceTaxonomy.VersionComparison.Controllers
+{
+    public class AdminController : Controller
```

Comment:
Do we need here  Authorize:Admin attribute?

Reply:
I don't believe so.